packages/catalog-search/src/tests/ClearCurrentRefinements.test.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
+import { renderWithRouter } from '@edx/frontend-enterprise-utils';
```

Comment:
Because `renderWithRouter` is only about rendering React components, I wonder if it should remain in this library? 
I could be convinced otherwise, thought.

Reply:
There will be 2 packages within the `frontend-enterprise` monorepo that export React components and both use `renderWithRouter`:

1. `@edx/frontend-enterprise-catalog-search`
2. `@edx/frontend-enterprise-logistration`

I opted to move `renderWithRouter` into `@edx/frontend-enterprise-utils` to be easily shared between libraries (or between our Enterprise MFEs).